test/cases/ostree-simplified-installer.sh

```diff
@@ -796,40 +796,43 @@ fi
 
 # TODO(runcom): change this to butane to check that too
 # also, write the very same test for ostree-raw-image
+BU_PATH="${HTTPD_PATH}"/butane
+sudo mkdir -p ${BU_PATH}
+BU_CONFIG_PATH="${BU_PATH}/bu_config.bu"
+sudo tee "$BU_CONFIG_PATH" > /dev/null << EOF
+variant: r4e
+version: 1.0.0
+ignition:
+  config:
+    merge:
+      - source: "http://192.168.100.1/ignition/sample.ign"
+  timeouts:
+    http_total: 30
+passwd:
+  users:
+    - name: core
+      password_hash: "\$6\$GRmb7S0p8vsYmXzH\$o0E020S.9JQGaHkszoog4ha4AQVs3sk8q0DvLjSMxoxHBKnB2FBXGQ/OkwZQfW/76ktHd0NX5nls2LPxPuUdl."
+      ssh_authorized_keys:
+        - "${SSH_KEY_PUB}"
+      groups:
+        - wheel
+EOF
+
 IGN_PATH="${HTTPD_PATH}/ignition"
 sudo mkdir -p ${IGN_PATH}
 IGN_CONFIG_PATH="${IGN_PATH}/config.ign"
-sudo tee "$IGN_CONFIG_PATH" > /dev/null << EOF
-{
-  "ignition": {
-    "config": {
-      "merge": [
-        {
-          "source": "http://192.168.100.1/ignition/sample.ign"
-        }
-      ]
-    },
-    "timeouts": {
-      "httpTotal": 30
-    },
-    "version": "3.3.0"
-  },
-  "passwd": {
-    "users": [
-      {
-        "groups": [
-          "wheel"
-        ],
-        "name": "core",
-        "passwordHash": "\$6\$GRmb7S0p8vsYmXzH\$o0E020S.9JQGaHkszoog4ha4AQVs3sk8q0DvLjSMxoxHBKnB2FBXGQ/OkwZQfW/76ktHd0NX5nls2LPxPuUdl.",
-        "sshAuthorizedKeys": [
-          "${SSH_KEY_PUB}"
-        ]
-      }
-    ]
-  }
-}
-EOF
+
+# Pull the latest release
+greenprint "Pulling butane's latest container image"
+sudo podman pull quay.io/coreos/butane:release
```

Comment:
I think this is pointless, because it pulls the image using *rootful* podman, whereas the actual container is run using *rootless* podman. IIRC they don't share the image storage.